Gemfile

```diff
@@ -29,7 +26,6 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'rspec-rails', '~> 5.0'
   gem 'simplecov'
-  gem 'webmock'
```

Comment:
not used

Reply:
Is it not needed for https://github.com/sul-dlss/gis-robot-suite/blob/main/spec/spec_helper.rb#L4

Reply:
@edsu this is dor-indexing-app;  the spec you mentino is in gis-robot-suite.  I'm confused.

Reply:
oops!!!